client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -260,6 +261,9 @@ function CellIcons({ support }: { support: bcd.SupportStatement | undefined }) {
   return (
     <div className="bc-icons">
       {supportItem.prefix && <Icon name="prefix" />}
+      {supportItem.notes?.length &&
+        !supportItem.version_removed &&
+        !supportItem.partial_implementation && <Icon name="footnote" />}
```

Comment:
I have a last suggestion for improvement:

You could extract a function in `./util.tsx` that takes a `supportItem` and decides whether the footnote icon should be shown. Then you can reuse it here and below.

Something like:

```ts
function hasNoteworthyNotes(support: bcd.SimpleSupportStatement) {
  return support.notes?.length &&
    !support.version_removed &&
    !support.partial_implementation;
}
```

And then this would be:

```suggestion
      {hasNoteworthyNotes(supportItem) && <Icon name="footnote" />}
```